packages/flutter_migrate/analysis_options.yaml

```diff
@@ -0,0 +1,19 @@
+# Specify analysis options.
```

Comment:
Why do you need non-standard options?

If this is really necessary, you'll need to allow it in script/configs/custom_analysis.yaml with an explanation of why.

Reply:
Discussing dependency issue with @christopherfujino . I'll remove the custom analysis for now.